### PR TITLE
soltest on windows

### PR DESCRIPTION
**Pull request by winsvega**
moved from https://github.com/ethereum/solidity/pull/948
(git crashed upon rebasing)

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -34,6 +34,8 @@ configuration:
 # We can re-enable it when we find a way to mitigate the unreliability
 # issues.  Have automated builds be reliable is the more important thing.
 #cache: build
+#init:
+#    - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
```
Comment (winsvega) [inline appveyor.yml:40]: in case we'd need a RDP detail to login into appveyor

File: test/RPCSession.h [hunk 2]
```diff
@@ -114,6 +114,7 @@ class RPCSession: public boost::noncopyable
 
 	std::string const& account(size_t _id) const { return m_accounts.at(_id); }
 	std::string const& accountCreateIfNotExists(size_t _id);
+	void setSleepTime(int _time) { m_sleepTime = _time; }
```
Comment (winsvega) [inline test/RPCSession.h]: otherwise mining timeout erros on appveyor

File: appveyor.yml (hunk 1 above)
Comment (chriseth) [reply appveyor.yml:40]: Please add that as a comment to the code and not only to the PR

File: scripts/install_eth.cmake [hunk 3]
```diff
@@ -0,0 +1,48 @@
+function(download URL DST_FILE STATUS)
```
Comment (bobsummerwill) [inline scripts/install_eth.cmake:29]: Please could you add the block file header, with the copyright/license, etc?
You can copy-and-tweak it from `install_deps.sh`.   Thanks!

File: test/RPCSession.cpp [hunk 4]
```diff
@@ -32,6 +32,34 @@
 using namespace std;
 using namespace dev;
 
+#if defined(_WIN32)
```
Comment (bobsummerwill) [inline test/RPCSession.cpp]: I don't know that it is worth us carrying this code around just for that one error message.

I assume this was useful to you while troubleshooting this implementation, but maybe doesn't have lasting value?

**Comment by bobsummerwill** [comment]
Generally, though, great!   Looks good barring the small points in the comments.

**Pull request merged**